### PR TITLE
9 New Paintings

### PR DESCRIPTION
**Pull request by EmoGarbage404**
## About the PR 
Added 9 new paintings and a random spawner.
The spawner does not include the moony painting or the empty frame.

The paintings are, in order of the image,
- empty frame
- Original composition
- The Persistence of Memory
- The Son of Man
- The Kiss
- The Scream
- The Great Wave off Kanagawa 
- Cafe Terrace at Night
- Nighthawks

**Screenshots**

image

**Changelog**

:cl:
- add: A variety of different paintings can now be found across the station.

### CODE REVIEW

**Review by moonheart08 — CHANGES_REQUESTED** [review]
Need copyright info for the minecraft painting.

**Comment by moonheart08** [comment]
My bad. So just to verify: Every painting transformed here (Except my pfp which I grant you permission to make abstract artwork out of)  is either public domain, or licensed under a license that you can relicense to CC-BY-SA-3.0?

**Comment by EmoGarbage404** [comment]
> My bad. So just to verify: Every painting transformed here (Except my pfp which I grant you permission to make abstract artwork out of) is either public domain, or licensed under a license that you can relicense to CC-BY-SA-3.0?

Yes.

**Comment by efzapa** [comment]
> Need copyright info for the minecraft painting.

I'll just paste what I said in the discord for time saving



Zetterstrand, being the one who made the minecraft artwork licensed his work under a (CC BY-NC-SA 4.0) meaning we can use his works as long as we give credit and don't use it for commercial purposes (that shouldn't be a problem as long as the game remains F2P)

ShareALike also means if we remix or edit it, which we will have to in order for it to fit the pixel theme, we have to use his same licence, also not a problem

tl;dr, we can use minecraft paintings as long as we use the same licence and have the game F2P

**Comment by EmoGarbage404** [comment]
> > Need copyright info for the minecraft painting.
> 
> I'll just paste what I said in the discord for time saving
> 
> Zetterstrand, being the one who made the minecraft artwork licensed his work under a (CC BY-NC-SA 4.0) meaning we can use his works as long as we give credit and don't use it for commercial purposes (that shouldn't be a problem as long as the game remains F2P)
> 
> ShareALike also means if we remix or edit it, which we will have to in order for it to fit the pixel theme, we have to use his same licence, also not a problem
> 
> tl;dr, we can use minecraft paintings as long as we use the same licence and have the game F2P

This isnt relevant as there are no minecraft paintings. This was already stated earlier in the pr conversation. Everything is fine, it just needs to be merged.
